src/Serilog.Settings.AppSettings/Serilog.Settings.AppSettings.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <Description>XML configuration (System.Configuration &lt;appSettings&gt;) support for Serilog.</Description>
-    <VersionPrefix>2.2.0</VersionPrefix>
+    <VersionPrefix>2.2.2</VersionPrefix>
```

Comment:
The version doesn't need to be update for -dev builds; I'll tweak this and merge now.